external-crates/move/crates/move-analyzer/prettier-move/tests/mega-test/misc.exp.move

```diff
@@ -0,0 +1,139 @@
+// options:
+// printWidth: 80
+
+/*
+ * @title Timelock
+ *
+ * @notice Locks any object with the store ability for a specific amount of time.
+ *
+ * @dev We do not provide a function to read the data inside the {Timelock<T>} to prevent capabilities from being used.
+ */
+module suitears::timelock {
+    fun calculate_pending_rewards<StakeCoin, RewardCoin>(
+        acc: &Account<StakeCoin, RewardCoin>,
+        stake_factor: u64,
+        accrued_rewards_per_share: u256,
+    ): u64 {
+        (
+            (
+                (
+                    (acc.amount as u256) * accrued_rewards_per_share /
+                    (stake_factor as u256),
+                ) -
+                acc.reward_debt,
+            ) as u64,
+        )
+    }
+
+    // === Imports ===
+
+    public fun lock<T: store>(
+        data: T,
+        c: &Clock,
+        unlock_time: u64,
+        ctx: &mut TxContext,
+    ): Timelock<T> {
+        // It makes no sense to lock in the past
+        assert!(unlock_time > c.timestamp_ms(), EInvalidTime);
+    }
+
+    public fun propose<DaoWitness: drop>(
+        dao: &mut Dao<DaoWitness>,
+        c: &Clock,
+        authorized_witness: Option<TypeName>,
+        capability_id: Option<ID>,
+        action_delay: u64,
+        quorum_votes: u64,
+        hash: String,
+        // hash proposal title/content
+        ctx: &mut TxContext,
+    ): Proposal<DaoWitness> {
+        assert!(action_delay >= dao.min_action_delay, EActionDelayTooShort);
+        assert!(quorum_votes >= dao.min_quorum_votes, EMinQuorumVotesTooSmall);
+        assert!(hash.length() != 0, EEmptyHash);
+
+        let start_time = c.timestamp_ms() + dao.voting_delay;
+
```

Comment:
Shouldn't we remove excessive empty lines (or at least have an option to do so)?